src/main/scala/junctions/crossing.scala

```diff
@@ -91,3 +90,46 @@ object ClockToSignal {
     c2s.io.signal_out
   }
 }
+
+object LevelCrossing {
+  class SynchronizerBackend[T <: Data](typ: T, sync: Int, _clock: Clock) extends Module(Some(_clock)) {
```

Comment:
What is the purpose of that besides making the user recognize that the individual bits won't arrive at the same time? The one place we use this for a UInt (for tile hartid), the value isn't expected to change during normal operation.